### PR TITLE
Apps: uninstall

### PR DESCRIPTION
**Pull request by jinroh**
This PR adds the possibility to remove/uninstall an application from the HTTP API and CLI.

Tests are on there way.

### CODE REVIEW

File: pkg/apps/installer.go [hunk 1]
```diff
@@ -121,6 +121,23 @@ func (i *Installer) Update() {
...
+	if state := i.man.State; state != Ready && state != Errored {
+		return nil, ErrBadState
+	}
+	if err := deleteManifest(i.ctx, i.man); err != nil {
+		return nil, err
+	}
+	if err := vfs.RemoveAll(i.ctx, i.appDir()); err != nil {
+		return nil, err
+	}
+	return i.man, nil
```
Comment (nono) [inline pkg/apps/installer.go]: IIRC, we said that we have to remove some permissions when uninstalling an app. Probably the reverse of `permissions.CreateAppSet`

Comment (jinroh) [inline pkg/apps/installer.go]: Looks like the `permissions.DestroyApp` method (called in `deleteManifest`) does the work. can you confirm @aenario ?

Comment (aenario) [inline pkg/apps/installer.go]: Yes, I changed it when introducing sub-permissions.

File: pkg/apps/installer.go [hunk 2]
```diff
@@ -121,6 +121,23 @@ func (i *Installer) Update() {
 	return
 }
 
+// Delete will remove the application linked to the installer.
+func (i *Installer) Delete() (*Manifest, error) {
```
Comment (aenario) [inline pkg/apps/installer.go]: [~nitpick~ troll] Installer is now a weird name for this class...

**Pull request merged**